### ISSUE TITLE
Ceiling alien

### ISSUE THREAD

**Issue by bibbybabby** [studio-minus/ppg-bugs-and-requests#2364]
**Feature description**
alien that sticks to a ceiling and has its tongue hanging below, if something touches it it grabs and pulls it, if its edible then yum yum if its not then spew out and avoid

**Feature purpose**
Entity variety

**Additional context**
half-life barnacle

**Comment by DindinYT37** [comment]
Yeah, that's just a barnacle. I wonder how this wasn't already made by anyone as a mod or contraption.

**Comment by Drogo-man** [comment]
Yes.
